systemtests/ansible/inventory/systemtests.inventory

```diff
@@ -6,7 +6,7 @@ namespace=enmasseci
 enable_rbac=True
 api_server=True
 register_api_server=True
-service_catalog=True
+service_catalog=False
```

Comment:
Perhaps add to your template "Ensure that the pull request does not include completely unrelated changes you have accidentally checked in / pushed" ;-)

Reply:
I was just testing reviewer quality ;) Thanks!